src/sentry/auth/providers/saml2.py

```diff
@@ -226,6 +226,8 @@ class SAML2Provider(Provider):
       state during setup. The attribute mapping should map the `Attributes`
       constants to the Identity Provider attribute keys.
     """
+    # Most SAML providers require the advanced SSO feature
+    required_feature = 'organizations:sso-adv'
```

Comment:
on second thought, what if we kept this named 'sso-saml2'

Reply:
Hmm, yeah, ok I think I agree with that.